drag_and_drop_v2/templates/html/js_templates.html

```diff
@@ -5,27 +5,33 @@
         width:{{ width_percent }}%;
         height:{{ height_percent }}%;">
```

Comment:
@bradenmacdonald I'm not 100% sure what you mean by `p.description`, but assuming that you're talking about referencing the

```html
<p class="sr">{{{ description }}}</p>
```

element, I don't think that's necessary: Since the elements that hold the `title` and `description` of a zone are children of the `div` that represents the zone, screen readers should automatically associate their text nodes with the `div`.